server/index.js

```diff
@@ -1,13 +1,17 @@
 const express = require('express');
 const mongoose = require('mongoose');
-require('dotenv').config();
+const passport = require('passport');
+require('dotenv').config({ path: './src/config/.env' });
```

Comment:
this line gives me error. Let's use the default .env location which is the root of server :)

Reply:
Done :)

Reply:
I also added some validation to the signup route and re-tested everything.